CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # [3.4.0](https://github.com/CrowdStrike/tailwind-toucan-base/compare/v3.3.1...v3.4.0) (2023-03-20)
```

Comment:
Not sure why this is up here... I wonder if the changelog got messed up at some point during our switch from semantic-release to changesets.  At any rate, after this merges, I'll move this down to where it's supposed to be 😅